doc/man/scons.xml

```diff
@@ -3351,12 +3421,11 @@ from SCons.Script import *
 <para>A &consenv; has an associated dictionary of
 <firstterm>&consvars;</firstterm>
 that are used by built-in or user-supplied build rules.
-&Consvar; naming must follow the same rules as for
+&Consvar; naming must follow the same rules as used for
```

Comment:
same rules as Python..

Reply:
ok.

Reply:
Actually, come to think of it, since they're only strings used as dictionary keys, is this actually even true?